server/src/main/java/com/linecorp/centraldogma/server/internal/storage/repository/git/DefaultChangesApplier.java

```diff
@@ -83,7 +86,7 @@ int doApply(Revision unused, DirCache dirCache,
                         hasChanges = !rawContent.equals(oldRawContent);
                     } else {
                         // Otherwise, compare the parsed JSON nodes.
-                        final JsonNode oldJsonNode = oldContent != null ? Jackson.readTree(oldContent) : null;
+                        final JsonNode oldJsonNode = toJsonNode(changePath, oldContent);
```

Comment:
Shouldn't we compare the raw content for JSON5 because a comment might be added or modified?